flask_praetorian/base.py

```diff
@@ -484,3 +550,111 @@ def pack_header_for_user(
             **custom_claims
         )
         return {self.header_name: self.header_type + ' ' + token}
+
+    def send_registration_email(self, user=None, template=None,
+                                subject=None,
+                                **kwargs):
+        """
+        Sends a registration email to a new user, containing a time expiring
+            token usable for validation.
+        :param: user:                     The user object to tie claim to
+                                          (username, id, email, etc)
+        :param: subject:                  The registration email subject
+                                          override.
+        :param: override_access_lifepsan: Anything used in `encode_jwt_token`,
```

Comment:
This should be added as a separate keyword argument. It would probably be best not to pass along kwargs from this method to `encode_jwt_token` as that usage isn't really clear in the API

Reply:
i was just hating on the 79 character pep limit at that time and gave up to move onto the next task

Reply:
submitted